test/support/span_parser.dart

```diff
@@ -106,11 +106,11 @@ class ScopeSpan {
 
   bool contains(int token) => (start <= token) && (token < end);
 
-  /// Splits the current [ScopeSpan] into multiple spans separated by [cond].
+  /// Splits the current [ScopeSpan] into multiple spans separated by [condition].
```

Comment:
The source for this file is in DevTools and will be synced to her periodically if fixes are made, so these changes would need to be contributed there to avoid being overwritten.

https://github.com/dart-lang/dart-syntax-highlight/blob/0df49894ac85b1b44d9120ebbcb01ab418974fea/test/support/span_parser.dart#L5-L8

Reply:
Oh I see, okay...